oceanbasecloud/assets/oauth_clients.json

```diff
@@ -5,10 +5,11 @@
     ],
     "client_role": "integration",
     "name": "oceanbasecloud",
-    "onboarding_url": "https://cloud.oceanbase.com/integrations/Datadog",
+    "onboarding_url": "https://console-en.oceanbase.com/integrations/Datadog",
```

Comment:
Hi Team, This link doesn't seem to load. Is it working on your end?

Reply:
Hi ,JoshPatel13 . This  domain address  was  temporary unavailable yesterday . It's recovered alreay by now.

Reply:
Sounds good Thanks!